selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -897,7 +894,7 @@ public void selectItemIntoAutocompleteAndPerformDoubleClick(String item) {
    */
   public void selectAutocompleteProposal(String item) {
     seleniumWebDriverHelper.waitAndClick(
-        By.xpath(format(AUTOCOMPLETE_CONTAINER + "/li/span[text()='%s']", item)));
```

Comment:
Could you, please, explain what `text()` was replaced by `.` in _xpath_ "/li/span[text()='%s']" for?

Reply:
. recursively takes the text in all subelements ("foo <span>bar</span>" is "foo bar", whereas in "text()" it woudl be "foo ".

Reply:
I see, thank you for explanation.